src/System.Runtime/src/ApiCompatBaseline.uap101aot.txt

```diff
@@ -0,0 +1,291 @@
+Compat issues with assembly System.Runtime:
+TypesMustExist : Type 'System.AccessViolationException' does not exist in the implementation but it does exist in the contract.
```

Comment:
Rather than baseline cases like this, should we instead just add the type to corefx?  (Unless we expect code in corert to actually throw/catch it, in which case it should be defined there.)